src/extension.ts

```diff
@@ -1,71 +1,27 @@
 import * as vscode from 'vscode';
-import * as io from 'socket.io-client';
 import axios from 'axios';
-import { StateService } from './services/stateService';
-import { NotificationService } from './services/notificationService';
-import { ProjectDataTreeProvider } from './views/project/projectsTreeDataProvider';
+import { StateService } from './services/storage/state.service';
+import { NotificationService } from './services/notification.service';
+import { ProjectDataTreeProvider } from './views/project/projects-tree-data-provider';
 import { Project } from './models/project.model';
 import { User } from './models/user.model';
-import { SignInDataTreeProvider } from './views/signIn/signInTreeDataProvider';
-import { DATABASE_COLUMNS } from './constants/strings';
+import { SignInDataTreeProvider } from './views/signIn/signIn-tree-data-provider';
+import { StorageKey } from './services/storage/storage-key';
 
 const apiEndPoint = 'http://localhost:1337';
-const socketEndPoint = 'http://localhost:1337';
 
-let socket: any = null;
 let state: StateService;
 let notificationService: NotificationService;
 let projectDataTreeProvider: ProjectDataTreeProvider;
 let signInTreeProvider: SignInDataTreeProvider;
 
-function initSocket() {
```

Comment:
Where is socket? we need socket integration in case user starts or stops an activity using other client apps.

Reply:
I'm going to postpone socket integration, as I remember there was problem for socket integration.

Reply:
Ok, But please add a warning in plugin description when adding plugin to vscode store about this. So users won't get confused with syncing problems between devices and clients.

Reply:
Sure